src/Transports.Subscriptions.Abstractions/Transports.Subscriptions.Abstractions.csproj

```diff
@@ -8,9 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="$(GraphQLVersion)" />
+    <PackageReference Include="GraphQL.NewtonsoftJson" Version="$(GraphQLVersion)" />
```

Comment:
I remember this dependency. We will deal with it later.

Reply:
Yea I think this falls under the other ticket I raised.